server/index.js

```diff
@@ -0,0 +1,37 @@
+const express = require('express');
+const mongoose = require('mongoose');
+const Rooms = require('../database/Room.js');
+
+const app = express()
+const PORT = 3000;
+
+app.use(express.json())
+
+app.get('/api/reviews/:roomid/', (req, res) => {
+  Rooms.find({id: req.params.roomid}, (err, room) => {
+    if (err) {
+      res.sendStatus(500);
+    } else {
+      //the user is able to request a sub-section of room reviews using a query param stylized as such: '?reviewgroup='.
+      var selReviewGroup = Number(req.query.reviewgroup);
+      var maxNumOfReviewsInAGroup = 5;
+      var selReviewGroupIsWithinBounds = (selReviewGroup + 1) * maxNumOfReviewsInAGroup <= room[0].reviews.length;
```

Comment:
You can use mongoose's findOne() to return a single room, so you don't have to grab it from an array of rooms with room[0].
See [https://mongoosejs.com/docs/api.html#model_Model.findOne](https://mongoosejs.com/docs/api.html#model_Model.findOne)

Also, if you want to use find() anyways, you can destructure out the first room like so,
`(err, [room]) => {room.reviews.length} `